src/it/scala/com/spark/redshift/community/AWSCredentialsInUriIntegrationSuite.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.databricks.spark.redshift
+package com.spark.redshift.community
```

Comment:
According to this: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

We should be using 

package com.spark_redshift_community

otherwise, this maybe thought as owned by "spark.com", which TIL is really not about Apache Spark :D

Reply:
I was thinking the same actually. will update, thanks for having a look!

Reply:
it's now `com.spark_redshift_community.spark.redshift`.
So it follows github org (com.spark_redshift_community) and project (spark-redshift)